functions/src/mappers/github/repository.mapper.ts

```diff
@@ -24,6 +30,9 @@ export interface GitHubRepositoryModel {
   pullRequests?: GitHubPullRequestModel[];
   events?: GitHubEventModel[];
   releases?: GitHubReleaseModel[];
+  issues?: GitHubIssueModel[];
+  updatedAt: firestore.Timestamp;
+  contributors: GitHubContributorModel[];
```

Comment:
Low priority, but could `updateAt` be at the end

Reply:
Done